cmd/cli/flags/config.go

```diff
@@ -0,0 +1,30 @@
+package flags
+
+import (
+	"github.com/solo-io/solo-kit/cmd/cli/options"
+	"github.com/spf13/pflag"
+)
+
+const (
+	input          = "input"
+	input_default  = "api/v1/"
+	output         = "output"
+	output_default = "pkg/api/v1"
+	docs           = "docs"
+	docs_default   = "doc/docs"
+	config         = "config"
+	gogo           = "gogo"
+	gogo_default   = true
```

Comment:
consider give the reader an idea of what these constants are going to be used for
e.g.:
```
inputPrefix          = "input" // check if a file has this string as a prefix
```